Grammar/python.gram

```diff
@@ -134,6 +134,7 @@ import_from[stmt_ty]:
         _Py_ImportFrom(NULL, b, _PyPegen_seq_count_dots(a), EXTRA) }
 import_from_targets[asdl_seq*]:
     | '(' a=import_from_as_names [','] ')' { a }
+    | invalid_import_from_targets
     | import_from_as_names
```

Comment:
I'd prefer it, if we had the invalid alternative as last, so that it doesn't negatively impact performance that much. Maybe something like:
```suggestion
    | import_from_as_names !','
    | invalid_import_from_targets
```